docs/resources/Guild.md

```diff
@@ -521,6 +521,111 @@ A partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object. Represents an Offl
+| default_channel_ids     | array of snowflakes                                                                                                     | channel ids that new members get opted into automatically      |
+| enabled                 | boolean                                                                                                                 | is guild onboarding enabled?                                   |
+
+
+###### Onboarding Prompt Structure
+
+| Field            | Type                                                                                                    | Description                                            |
+| ---------------- | ------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
+| id               | snowflake                                                                                               | the id of the prompt                                   |
+| options          | array of [prompt option](#DOCS_RESOURCES_GUILD/guild-onboarding-object-prompt-option-structure) objects | the options available to the prompt                    |
+| title            | string                                                                                                  | the title of the prompt                                |
+| single_select    | boolean                                                                                                 | can only one option of the prompt be selected at time? |
+| required         | boolean                                                                                                 | is this prompt required in onboarding flow?            |
+| in_onboarding    | boolean                                                                                                 | is this prompt in the onboarding flow?                 |
+| type             | [prompt type](#DOCS_RESOURCES_GUILD/guild-onboarding-object-prompt-type)                                | the type of the prompt                                 |
+
+
+###### Prompt Option Structure
+
+| Field       | Type                                               | Description                                                                               |
+| ----------- | -------------------------------------------------- | ---------------------------------------------------- |
+| id          | snowflake                                          | the id of the option                                 |
+| channel_ids | array of snowflakes                                | the channels opted into when this option is selected |
+| role_ids    | array of snowflakes                                | the roles assigned when this option is selected      |
+| emoji       | [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | the emoji of the option                              |
```

Comment:
The docs for the [emoji structure](https://discord.com/developers/docs/resources/emoji#emoji-object-emoji-structure) state that the `name` can only be null for reactions. However, when there is no emoji set for a prompt option the `emoji` is `{ "id": null, "name": null, "animated": false }`. Should the docs be updated for the emoji structure to reflect this?

Reply:
Ideally it should be `emoji: null` then instead of a structure filled with nulls; I think that's how it works in other parts of the API too.